### ISSUE TITLE
Every X seconds cron fails with maximum call stack exceeded

### ISSUE THREAD

**Issue by fschaeffler** [Hexagon/croner#260]
**Describe the bug**
When defining a cron for every X seconds, I get an exception with maximum call stack exceeded.

**To Reproduce**
1. install croner
2. require croner
3. create a cron which should trigger every X seconds

Example: 
```
const { Cron } = require('croner')

// works as expected
new Cron('* * * * * *', () => console.log('test every second'))

// fails with maximum call stack exceeded
new Cron('/3 * * * * *', () => console.log('test 3 seconds'))

Uncaught RangeError: Maximum call stack size exceeded
    at CronDate.findNext (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:15327)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16435)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16791)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
    at CronDate.recurse (/home/flo/dev/private/trading/autotrader/node_modules/croner/dist/croner.min.cjs:1:16993)
```

**Expected behavior**
There's no exception when creating this cron and it gets triggered every X seconds.

**System:**
 - OS: Linux (`Linux 6.8.0-45-generic #45-Ubuntu SMP PREEMPT_DYNAMIC Fri Aug 30 12:02:04 UTC 2024 x86_64 GNU/Linux`)
 - Runtime: Node.js
 - Version: 18.20.4
 - croner: 8.1.2

**Comment by Hexagon** [comment]
Interesting! What if you add a * before /3, like `*/3 * * * * *`?

**Comment by Hexagon** [comment]
Should work, but leave this open. I need to fiure out if it should be handled as a syntax error, or "just work". Call stack is not expexted behavior!

**Comment by fschaeffler** [comment]
You're totally right. Adding an asterisk before it solves the issue.

**Comment by Hexagon** [comment]
Resolved in dev channel, version `9.0.0-dev.4`

**Comment by Hexagon** [comment]
Now available in stable channel, through `croner@9.0.0`
